src/PDO/Peachpie.Library.PDO/PDOStatement.cs

```diff
@@ -443,6 +443,11 @@ public virtual PhpArray fetchAll(PDO_FETCH fetch_style = default, PhpValue fetch
                 }
             }
 
+            if (fetch_style.HasFlag(PDO_FETCH.FETCH_CLASS) && !fetch_argument.IsEmpty)
```

Comment:
Instead of `fetch_style`, we should probably use `style` (below) since it might fallback to `_default_fetch_type` (previously set default)

The same would apply for `FETCH_COLUMN` above. Or does PHP respect the default value in this case at all?

Reply:
By default value, I guess you mean if there are any differences between:
```php
$pdo->fetchAll(0);
// vs.
$pdo->fetchAll();
```

Reply:
There are absolutely no differences between these two. PHP must think `0` is `falsy`...

Reply:
I will move the `FETCH_COLUMN` test to use `style`